sys-libs/llvm-libunwind/files/Makefile

```diff
@@ -0,0 +1,43 @@
+ASFLAGS+=-Wa,--noexecstack
```

Comment:
Now I dropped my Makefile and switched to cmake, with a downstream patch. This patch needs more work to be upstreamed though.